packages/web/integration-tests/tests/errors/views/minified_file_errors.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Errors test</title>
+  <%- renderAgent({}) %>
+</head>
+<body>
+    <h1>Minified files with source map ids</h1>
+    <button id="button1">Button 1</button>
+    <button id="button2">Button 2</button>
+    <script src="/integration-tests/tests/errors/views/script1.min.js"></script>
+    <script src="/integration-tests/tests/errors/views/script2.min.js"></script>
```

Comment:
Should also add a test for `<script type="module">` as [it doesn't set document.currentScript](https://developer.mozilla.org/en-US/docs/Web/API/Document/currentScript) and the URL should be gotten from [`import.meta.url` there](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import.meta)